### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by ochococo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,11 +46,11 @@ Example:
 ```json
 {
 	"timestamp": 1485778030,
-	"readings": [
+	"readings": {
```
Comment (MaciejGad) [inline README.md]: what do you think about only one measurement with one `timestamp`, so `readings` will be an object not an array of objects:
```json
{
"timestamp": 1485778030,
"readings": {
  "PM25": 201.1,
  "PM10": 102.0
  }
}
```
and if we want to send multiple measurements at once, we could send an array:
```json
[
	{
	"timestamp": 1485778030,
	"readings": {
	  "PM25": 201.1,
	  "PM10": 102.0
	  }
	},
	{
	"timestamp": 1485778031,
	"readings": {
	  "PM25": 202.1,
	  "PM10": 103.0
	  }
	}
]
```

Comment (ochococo) [inline README.md]: This is actually a great idea!

**Comment by ochococo** [comment]
@MaciejGad ✨

**Comment by MaciejGad** [comment]
@ochococo +1

**Pull request merged**